src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseTaskDataObject.cs

```diff
@@ -286,9 +344,17 @@ internal set
         public string targetDbName = string.Empty;
 
         /// <summary>
-        /// The database used to restore from
+        /// The database from the backup file used to restore to by default
         /// </summary>
-        public string sourceDbName = string.Empty;
+        public string DefaultDbName
+        {
+            get
+            {
+                var dbNames = GetSourceDbNames();
+                string dbName = dbNames.First();
```

Comment:
dbNames will never be null or an empty list?  If that's not a true statement we should due null checking and perhaps use FirstOrDefault